reasoner/planner/OrderingCoster.java

```diff
@@ -100,8 +100,8 @@ LocalAllCallsCosting createAllCallsCosting(ReasonerPlanner.CallMode callMode, Li
             } else {
                 disconnectedCost += resolvableCost;
             }
-            // TODO: Add permutative traversal cost as below. (Needs to be uncommented and tested)
-            // acyclicCost += estimator.answerEstimate(boundVars);
+            // Traversal cost - DFS style permutative work
+            acyclicCost += estimator.answerSetSize() * RELATIVE_COST_ANSWER_COMBINATION;;
```

Comment:
Add traversal cost component to the all-calls costing.
Without this, the flat planner would pick plans fine, but the global planner would not be able to tell the difference between a good flat-plan which avoided this and one that didn't.